cmd/pebble/cmd_help.go

```diff
@@ -167,16 +167,20 @@ type helpCategory struct {
 // helpCategories helps us by grouping commands
 var helpCategories = []helpCategory{{
 	Label:       "Basics",
-	Description: "basic management",
-	Commands:    []string{"start", "stop", "autostart"},
+	Description: "basic service management",
+	Commands:    []string{"autostart", "run", "services", "start", "stop"},
+}, {
+	Label:       "Configuration",
+	Description: "manage configuration",
+	Commands:    []string{"add", "plan"},
 }, {
 	Label:       "Changes",
-	Description: "basic management",
-	Commands:    []string{"history", "tasks", "abort"},
+	Description: "view changes, tasks, and warnings",
+	Commands:    []string{"changes", "okay", "tasks", "warnings"},
```

Comment:
Warnings (warnings, okay) are global concepts, and not just about changes/tasks. It doesn't feel like a coherent group. The order is also strange as it mixes the two ideas (changes, warnings, changes, warnings), but I think we should separate those two for the groups to make more sense together.

Reply:
Sounds good - I've moved to the "Other" category (same as `snap help` does).